client/client_pool/src/concord_client_pool.cpp

```diff
@@ -408,7 +411,10 @@ void SingleRequestProcessingJob::execute() {
     write_config_.request.span_context = span_context_;
     write_config_.request.pre_execute = flags_ & PRE_PROCESS_REQ;
     res = processing_client_->SendRequest(write_config_, std::move(request_));
-    if (callback_) callback_(res);
+    if (callback_) {
+      LOG_INFO(KEY_EX_LOG, "got callback sending response");
```

Comment:
Why KEY_EX_LOG?

Reply:
Since the SingleRequestProcessingJob class has no logger, I'm just testing it here for now. After I confirm it's working, I will remove this log which has no purpose.